account-management-api/src/main/java/uk/gov/di/accountmanagement/lambda/MFAMethodsCreateHandler.java

```diff
@@ -94,6 +94,9 @@ public APIGatewayProxyResponseEvent mfaMethodsHandler(
                     case INVALID_PRIORITY_IDENTIFIER -> {
                         return generateApiGatewayProxyErrorResponse(400, ErrorResponse.ERROR_1001);
                     }
+                    case AT_MAXIMUM_MFA_COUNT -> {
```

Comment:
I'd like to avoid using 'count' terminology here, as I read it as being about mfa attempt counts. Maybe "BACKUP_AND_DEFAULT_METHOD_ALREADY_EXIST"? (and same for elsewhere where we call this count)